filebeat/tests/integration/journald_test.go

```diff
@@ -0,0 +1,120 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Elasticsearch B.V. licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+//go:build integration && linux
+
+package integration
+
+import (
+	"context"
+	_ "embed"
+	"encoding/json"
+	"fmt"
+	"os/exec"
+	"path/filepath"
+	"syscall"
+	"testing"
+	"time"
+
+	"github.com/google/uuid"
```

Comment:
Use `github.com/gofrs/uuid`. See https://github.com/elastic/beats/pull/40161 for context.

https://github.com/elastic/beats/blob/f9b41f221db70bb2ef1f17b7e2895c1a9d01f9ba/.golangci.yml#L123-L127

Reply:
Thanks! I'm a bit puzzled why the lint didn't spot that :thinking: